packages/client/package.json

```diff
@@ -41,7 +41,8 @@
     "prepare": "npm run build",
     "build": "npm run build:clean && tsc",
     "build:clean": "shx rm -rf ./dist",
-    "lint": "tslint --project .",
+    "lint": "prettier 'src/**/*.ts' --check && eslint --ext .ts src",
+    "prettier-fix": "prettier --write 'src/**/*.ts'",
```

Comment:
nice! maybe we rename this to `lint-fix` and make it run eslint with the fix flag too.